config/webpack/rscWebpackConfig.js

```
@@ -0,0 +1,111 @@
+          testValue === 'style-loader'
+        );
+      });
+      const cssLoader = rule.use.find((item) => {
+        const testValue = typeof item === 'string' ? item : item?.loader;
+        return testValue?.includes('css-loader');
+      });
+      if (cssLoader?.options?.modules) {
+        cssLoader.options.modules = { ...cssLoader.options.modules, exportOnlyLocals: true };
+      }
+    } else if (rule.use && (rule.use.loader === 'url-loader' || rule.use.loader === 'file-loader')) {
+      rule.use.options.emitFile = false;
```

ESLint

[error] Assignment to property of function parameter 'rule'.

(no-param-reassign)